massa-ledger-exports/src/ledger_changes.rs

```diff
@@ -526,6 +526,30 @@ impl LedgerChanges {
         self.0.get(addr)
     }
 
+    /// Retrieves all the bytcode updates contained in the current changes
+    pub fn get_bytecode_updates(&self) -> Vec<Bytecode> {
+        let mut v = Vec::new();
+        for (_addr, change) in self.0.iter() {
+            match change {
+                SetUpdateOrDelete::Set(LedgerEntry { bytecode, .. }) => {
+                    // IMPORTANT TODO: investigate on why the len check is needed
```

Comment:
If we create a new address then a `Set` will be created with all fields filled and bytecode will be empty.

Reply:
The comment has been updated